kb/temporal-platform-limits-sheet.md

```diff
@@ -13,11 +13,14 @@ Here is a list of many hard (error) or soft (warn) limits that you could encount
 
 <!-- truncate -->
 
+- **Identifiers**: By default, the maximum length for identifiers (like Workflow Id, Workflow Type, and Task Queue name) is 1000 characters.
+  - This is configurable with the `limit.maxIDLength` dynamic config, which is set to 255 in [our SQL example](https://github.com/temporalio/docker-compose/blob/93d382ef9133e4cde8ce311de5153cd0cc9fbd0c/dynamicconfig/development-sql.yaml#L1-L2).
```

Comment:
Not fan of linking to lines in source - seen these change a lot.
Maybe link to overall file?
In the not so distant future we should be sporting a Dynamic Config ref will some of these knobs on there.

Reply:
This link is to a specific commit so that it won't change.